expression/aggregation.go

```diff
@@ -243,7 +243,7 @@ func (sf *sumFunction) GetStreamResult() (d types.Datum) {
 		sf.streamCtx = &ast.AggEvaluateContext{}
 	}
 	d = sf.streamCtx.Value
-	sf.streamCtx = &ast.AggEvaluateContext{}
+	sf.streamCtx = nil
```

Comment:
you set `sf.streamCtx = &ast.AggEvaluateContext{}` then `sf.streamCtx = nil` ???